bench/.editorconfig

```diff
@@ -1669,6 +1718,74 @@ dotnet_diagnostic.CA5404.severity = none
 # Help Link: https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca5405
 dotnet_diagnostic.CA5405.severity = none
 
+# Title    : Use source generated logging methods for improved performance
+# Category : Performance
+dotnet_diagnostic.EA0000.severity = warning
+
+# Title    : Perform message formatting in the body of the logging method
+# Category : Performance
+# Help Link: https://TODO/ea0001
```

Comment:
NIT: I'm assuming we have an issue tracking these ToDos?

Reply:
https://github.com/dotnet/extensions/issues/4246